dev/NavigationView/NavigationView.cpp

```diff
@@ -2173,9 +2177,12 @@ void NavigationView::OnNavigationViewItemKeyUp(const winrt::IInspectable& sender
 
 void NavigationView::OnNavigationViewItemKeyDown(const winrt::IInspectable& sender, const winrt::KeyRoutedEventArgs& args)
 {
-    if (auto nvi = sender.try_as<winrt::NavigationViewItem>())
+    if (args.Key() != winrt::VirtualKey::Enter && args.Key() != winrt::VirtualKey::Space)
```

Comment:
If we keep the KeyUp handling, we should add a check here for `args.OriginalKey() == winrt::VirtualKey::GamepadA` as well

Reply:
Shouldn't it be `args.OriginalKey() != winrt::VirtualKey::GamepadA` since we are handling GamepadA in KeyUp?

Reply:
You are right, typo on my part. Thanks!